barcodes_generator_product/wizard/multi_barcode_wizard.py

```diff
@@ -0,0 +1,63 @@
+# Copyright (C) 2022 Akretion (<http://www.akretion.com>).
+# @author Kévin Roche <kevin.roche@akretion.com>
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from odoo import api, fields, models
+
+
+class BarcodeMultipleGeneratorWizard(models.TransientModel):
+    _name = "barcode.multiple.generator.wizard"
+    _description = "Barcode Multiple Generator Wizard"
+
+    product_ids = fields.Many2many(
+        comodel_name="product.product",
+        string="Products",
+    )
+    barcode_rule_id = fields.Many2one(
+        string="Barcode Rule",
+        readonly=False,
+        comodel_name="barcode.rule",
+        domain=[
+            ("generate_model", "=", "product.product"),
+            ("generate_automate", "=", True),
+        ],
+    )
+    barcode_base = fields.Integer(
+        readonly=False,
+        string="Barcode Base start with",
+    )
+    generate_type = fields.Selection(
+        string="Generate Type",
+        related="barcode_rule_id.generate_type",
+        readonly=True,
+    )
+    product_barcode_exception = fields.Html(
+        readonly=True, compute="_compute_product_barcode_exception"
+    )
+
+    @api.depends("product_ids", "product_ids.barcode", "barcode_rule_id")
```

Comment:
```suggestion
    @api.depends("product_ids.barcode")
```

Is enought dont you think ?